lib/files/file_manager.ts

```diff
@@ -238,6 +237,15 @@ export class FileManager {
           logger.info('removed ' + file);
         }
       }
-    })
+    });
+
+    let updateConfig = path.resolve(outputDir, 'update-config.json');
+    try {
+      fs.statSync(updateConfig);
```

Comment:
Hmmm... good question. I guess if the file doesn't exist, fs.unlinkSync will throw an error. Removing.